.github/workflows/build_scheduled.yml

```diff
@@ -0,0 +1,371 @@
+                const regionToAmi = build["artifact_id"].split(",");
+                regionToAmi.forEach((regionToAmi, i) => {
+                    const [region, ami] = regionToAmi.split(":");
+                    toPrint[i] = toPrint[i] + ` ${ami} |`;
+                });
+            });
+            
+            const header = [
+                "## AWS",
+                "",
+                "| AWS Region       | AMI ID (ARM64)          | AMI ID (x86_64)         |",
+                "|------------------|-------------------------|-------------------------|",
+            ]
+            fs.writeFileSync("./body.md", header.join("\n") + "\n" + toPrint.join("\n"));
+
+      - name: Download Google Cloud manifest
+        uses: actions/download-artifact@v4
+        with:
+          name: manifest_gcp.json
+
+        # The GCP manifest file look like this:
+        # "builds": [
+        #   {
+        #     "name": "spacelift",
+        #     "builder_type": "googlecompute",
+        #     "build_time": 1700479054,
+        #     "files": null,
+        #     "artifact_id": "spacelift-worker-us-1700478848-305dsvij",
+        #     "packer_run_uuid": "cdc82943-986b-5ab9-6ce1-9024ca0ebb6a",
+        #     "custom_data": null
+        #   },
+        #   {
+        #     "name": "spacelift",
+        #     "builder_type": "googlecompute",
+        #     "build_time": 1700479263,
+        #     "files": null,
+        #     "artifact_id": "spacelift-worker-eu-1700478848-305dsvij",
+        #     "packer_run_uuid": "2440c9b1-a342-3606-2661-6e5389bdffc6",
+        #     "custom_data": null
+        #   }
+        # ]
+  
+      - name: Write Azure and GCP AMI IDs to the markdown file
+        uses: actions/github-script@v7
+        env:
+          AZURE_VERSION: ${{ needs.azure.outputs.azure_version }}
+        with:
+          script: |
+            const fs = require("fs");
+    
+            content = fs.readFileSync("./manifest_gcp.json", "utf8");
+            manifest = JSON.parse(content);
+    
+            const gcpLinesToPrint = [];
+    
+            manifest["builds"].forEach((build) => {
+                    artifact = build["artifact_id"];
+                    if (artifact.indexOf("-us-") > 0) {
+                        gcpLinesToPrint.push(` - United States | \`${artifact}\``);
+                    }
+                    if (artifact.indexOf("-eu-") > 0) {
+                        gcpLinesToPrint.push(` - Europe | \`${artifact}\``);
+                    }
+                    if (artifact.indexOf("-asia-") > 0) {
+                        gcpLinesToPrint.push(` - Asia | \`${artifact}\``);
+                    }
+            });
+    
+            azureLines = [
+                "## Azure",
+                "",
+                "- Community Gallery Name | `spacelift-40913cda-9bf9-4bcb-bf90-78fd83f30079`",
+                "- Image name | `spacelift_worker_image`",
+                `- Version | \`${process.env.AZURE_VERSION}\``,
```

Comment:
Is it definitely an env var and not an output? https://github.com/spacelift-io/spacelift-worker-image/pull/65/files#diff-f34f634b79243aba35d2cb6ba211d4662ef39b7d812b12c3462feb1ce4c6de8aR148

Reply:
Ah right - fair enough.